testing/src/test/java/io/stargate/it/http/graphql/graphfirst/GraphqlFirstClient.java

```diff
@@ -51,8 +51,16 @@ public GraphqlFirstClient(String host, String authToken) {
    * @return the resulting version.
    */
   public UUID deploySchema(String namespace, String contents) {
+    return deploySchema(namespace, null, contents);
+  }
+
+  public UUID deploySchema(String namespace, UUID expectedVersion, String contents) {
```

Comment:
Nit: I've generally stuck with plain strings for method arguments in this class, because some of the tests call them with malformed input, e.g. a string that is not a UUID.
That's not the case for this method (yet?), but I think it would be nice to keep things uniform.